codec-agnostic-rtp-payload-format.md

```diff
@@ -43,27 +43,37 @@ informative:
   RFC6464:
   RFC6465:
   RFC6904:
+  RFC6184:
   SFrame:
+    target: https://tools.ietf.org/html/draft-omara-sframe
+    title: Secure Frame (SFrame)
+  WebRTCInsertableStreams:
+    target: https://w3c.github.io/webrtc-insertable-streams
+    title: WebRTC Insertable Media using Streams
 
 --- abstract
 
-
+WebRTC Media Chains usually rely on piping encoder output directly to packetizers. Media packetization formats often support a specific codec format and optimize RTP packets generation accordingly.
+With the development of Selective Forward Unit (SFU) solutions, WebRTC Media Chains are increasingly relying on application-specific transforms that seat between encoder and packetizer on one end and between depacketizer and decoder on the other end. These transforms are typically protecting media content from being accessible from the SFU, for instance using {{SFrame}} or {{WebRTCInsertableStreams}}.
+In that context, RTP packetizers can no longer expect to use packetization formats that mandate media content to be in a specific codec format.
+This document provides a solution to that problem by describing a generic RTP packetization format that can be used on any media content, and how to negotiate use of this format.
+This document also describes a solution to allow SFUs to continue performing packet routing on top of this generic RTP packetization format.
 
 --- middle
 
 Introduction
 ============
 
-The objective of this spec is to create a generic RTP packetization format that can be used with any audio or video codec or encrypted content and that allows SFUs to perform routing and layer selection without requiring access to the codec payload.
+As per Figure 1 of {{RFC7656}}, a Media Packetizer transforms a single Encoded Stream into one or several RTP packets.
+The Encoded Stream is coming straight from the Media Encoder and is expected to follow the format produced by the Media Encoder.
+A number of Media Packetizer formats have been designed to process a specific format produced by Media Encoder.
+For instance {{6184}} is dedicated to the processing of content produced by H.264 Media Encoders, and generates packets following NALUs organization.
 
-Media packetization and depacketization
-=======================
+As described in Figure 1 and Figure 2, WebRTC Media Chains are increasingly inserting application-specific Media Transformers between Media Encoder and Media Packetizer on the sending side, and between Media Depacketizer and Media Decoder on the receiving side.
+In browsers, these transformers are for instance implemented as JavaScript code injected using {{WebRTCInsertableStreams}}.
```

Comment:
In RTP the concept of browser is not defined, better just mention the w3c specs.

Reply:
Since we are talking of WebRTC, mentioning browsers, JS and WebRTCInsertableStreams makes sense to me.
This shows that the transforms can do arbitrary things.

Reply:
I kept it for now, let me know your thoughts